### ISSUE TITLE
css content alt text is ignored

### ISSUE THREAD

**Issue by spike-rabbit** [dequelabs/axe-core#4763]
### Product

axe-core

### Product Version

4.10.3

### Latest Version

- [x] I have tested the issue with the latest version of the product

#### Expectation
I want to use the alt text for the content attribute as described here: https://developer.mozilla.org/en-US/docs/Web/CSS/content
I would expect that axe detects the alt text.

#### Actual
Axe ignores the alt text.

#### How to Reproduce
Open in a browser and scan with axe browser extension

```html
<html>
  <style>
    .with-content {
      content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100' height='100'%3E%3Ccircle cx='50' cy='50' r='40' fill='blue' /%3E%3C/svg%3E") /
        "Alt text not detected"
    }
  </style>
  <body>
    <div class="container">
      <button class="with-content"></button>
    </div>
  </body>
</html>
```

Will report an issue, but is valid. At least chrome and FF show the alt text in their a11y trees.

#### Additional context
I opened a similar issue for playwright: https://github.com/microsoft/playwright/issues/35737

**Comment by straker** [comment]
Thanks for the issue. I tested this in different screen readers and browsers and this is the results:

* VoiceOver / Safari - "Button, main"
* Chrome / Jaws - "Alt text not detected, button"
* Firefox / NVDA - "Alt text not detected, button"

So it appears that VoiceOver / Safari doesn't support the CSS content alt text feature yet. This means axe-core can't pass the button as having an accessible name since it doesn't meet our [accessibility supported criteria](https://github.com/dequelabs/axe-core/blob/develop/doc/accessibility-supported.md#accessibility-supported). Once VoiceOver supports this we can revisit passing.

**Comment by spike-rabbit** [comment]
Thanks for the clarification. I filled a bug for webkit, as it seems to work with Chrome and VoiceOver.
See https://bugs.webkit.org/show_bug.cgi?id=292653

**Comment by straker** [comment]
@spike-rabbit A coworker did some further testing and we don't think it's fully fixed in Webkit. I'll post their findings:

```
<style>
    .with-content {
      content: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><circle cx="8" cy="8" r="8" fill="blue"/></svg>') / "alt for with-content";
    }
    .with-before-content::before {
      content: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><circle cx="8" cy="8" r="8" fill="blue"/></svg>') / "alt for with-before-content";
    }
  </style>

  <button id="a" class="with-content"></button>
  <span id="b" class="with-content"></span>
  <button id="c" class="with-before-content"></button>
```

* Chrome 138.0.7204.101: `#a` doesn't render a visual blue circle, `#b` and `#c` do. `#a` and `#b` report with accessible name "alt for with-content", `#c` reports with "alt for with-before-content"
* Safari 18.5 (20621.2.5.11.8): `#a` doesn't render a visual blue circle, `#b` and `#c` do. `#a` and `#b` report with no accessible name. `#c` reports with accessible name "alt for with-before-content".
* Firefox 140.0.4: Same as chrome

It looks like Webkit does not render the alt when the element itself uses the `content`, but does when it's on the pseudo element.

**Comment by spike-rabbit** [comment]
@straker this should be fixed now. I am just not sure when this is going to land in Safari. Maybe you know?
